bundles/org.openhab.core.karaf/.classpath

```diff
@@ -25,5 +25,6 @@
 			<attribute name="annotationpath" value="target/dependency"/>
 		</attributes>
 	</classpathentry>
+	<classpathentry kind="src" path="/org.openhab.core.config.discovery.addon"/>
```

Comment:
This looks pretty weird in here. Is this required? Isn't Maven correctly adding the bundle as a dependency to the classpath?

Reply:
> Isn't Maven correctly adding the bundle as a dependency to the classpath?

TBH I don't know if it is Maven or Eclipse which is messing with the classpath files :( ..it is not me :)